manager/integration/tests/longhorn.py

```diff
@@ -193,8 +195,8 @@ class GdapiClient(object):
     def __init__(self, access_key=None, secret_key=None, url=None, cache=False,
                  cache_time=86400, strict=False, headers=HEADERS, **kw):
         self._headers = headers
-        self._access_key = access_key
-        self._secret_key = secret_key
+        self._access_key = access_key.__str__()
```

Comment:
This is used to fix the following warnings:

```
- DeprecationWarning: Non-string usernames will no longer be supported in Requests 3.0.0.
Please convert the object you've passed in (None) to a string or bytes object in the near future to avoid problem
    
- DeprecationWarning: Non-string passwords will no longer be supported in Requests 3.0.0.
Please convert the object you've passed in (None) to a string or bytes object in the near future to avoid problems.
```
And  `access_key` & `secret_key` is set by default to `None`

Reply:
Can we set them to "" by default then? as well as the URL.

Reply:
fixed.